spring-vault-core/src/main/java/org/springframework/vault/core/VaultPkiOperations.java

```diff
@@ -108,4 +109,31 @@ enum Encoding {
 
 	}
 
+	/**
+	 * Retrieves the specified issuer's certificate. Includes the full ca_chain of the
+	 * issuer.
+	 * @param issuer reference to an existing issuer, either by Vault-generated
+	 * identifier, or the name assigned to an issuer. If empty or {@literal null}, the
+	 * issuer will default to the literal string default to refer to the currently
+	 * configured default issuer.
+	 * @return the {@link VaultIssuerCertificateRequestResponse} containing a
+	 * {@link org.springframework.vault.support.Certificate}
+	 * @see <a href=
+	 * "https://www.vaultproject.io/api/secret/pki/#read-issuer-certificate">GET *
+	 * /pki/issuer/:issuer_ref/json</a>
+	 *
+	 */
+	VaultIssuerCertificateRequestResponse getIssuerCertificate(@Nullable String issuer) throws VaultException;
+
+	/**
+	 * Retrieves the specified issuer's certificate. Includes the full ca_chain of the
+	 * issuer.
+	 * @return {@link java.io.InputStream} containing the encoded certificate or
+	 * {@literal null}
+	 * @see <a href=
+	 * "https://www.vaultproject.io/api/secret/pki/#read-issuer-certificate">GET
+	 * /pki/issuer/:issuer_ref/{der, pem}</a>
+	 */
+	InputStream getIssuerCertificate(@Nullable String issuer, Encoding encoding) throws VaultException;
```

Comment:
We should not allow `null` values for the `issuer`. See the comment above for more details.

Reply:
Updated the code